posts/models.py

```diff
@@ -1,17 +1,36 @@
 from django.db import models
 from users.models import User
+from multiselectfield import MultiSelectField
 # Create your models here.
 
 
+class Ingredients(models.Model):
+    name = models.CharField(max_length=120, blank=False)
+
+
 class Post(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    INGREDIENT_CHOICES = (
+        ('tomato', 'Tomato'),
+        ('eggplant', 'Eggplant'),
+        ('celery', 'Celery'),
+        ('egg', 'Egg'),
+        ('milk', 'Milk'),
+        ('fish', 'Fish'),
+        ('chicken', 'Chicken'),
+        ('oil', 'Oil')
+    )
+    # picked_ingredients
+    ingredients = models.ManyToManyField(Ingredients)
+    choices = MultiSelectField(choices=INGREDIENT_CHOICES, max_length=1000, blank=True)
 
-    # user = models.ForeignKey(User, on_delete=models.CASCADE)
+    rating = models.FloatField(max_length=10, default=0)
```

Comment:
You should not store average rating in `Post` model. Let's remove `rating` field. It is similar to `like count`, we talked about it with @yigitguler.